docs/backup-restore.md

```diff
@@ -3,30 +3,63 @@
 Since Scalar DB provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
 This document sets out some guidelines for backing up and restoring the databases that Scalar DB supports.
 
-## Cassandra
+## Scalar DB server
```

Comment:
Why the first section of the backup and restore guide is `Scalar DB Server`. It's pretty weird.
Please think about the structure before writing sentences.